ipynb/20190501_homogeneous_Liouville.py

```diff
@@ -1,224 +0,0 @@
-
-# coding: utf-8
-
-# In[ ]:
-
-
-from pyspecdata import *
-from scipy.linalg import expm
-
```

Comment:
I deleted the ipynb guys b/c they have similar functions defined to the main code, which is confusing.